conf/sidecar.yaml

```diff
@@ -215,6 +222,7 @@ cassandra_input_validation:
     - system_auth
     - system_views
     - system_virtual_schema
+    - sidecar_internal
```

Comment:
even though we've had `sidecar_internal` in the default list https://github.com/apache/cassandra-sidecar/blob/trunk/server/src/main/java/org/apache/cassandra/sidecar/config/yaml/CassandraInputValidationConfigurationImpl.java#L43 , we are actually missing it in the yaml. So added it here

Reply:
I ran into issues while manually testing this feature, so I've added it